benchmarks/30k_methods.rb

```diff
@@ -120009,7 +120009,7 @@ def inc()
 
 require 'harness'
 
-INTERNAL_ITRS = ENV.fetch("INTERNAL_ITRS", 2000).to_i
+INTERNAL_ITRS = Integer ENV.fetch("INTERNAL_ITRS", 2000)
```

Comment:
trivial: I wonder why you sometimes put parenthesis for `Kernel#Integer` and sometimes don't. A method name starting with a capital letter is already confusing, so I personally prefer always put parenthesis. WDYT?
```suggestion
INTERNAL_ITRS = Integer(ENV.fetch("INTERNAL_ITRS", 2000))
```

Reply:
I usually add parens for `Integer()`, indeed. I think here I was just trying to minimize the diff and noise, but I'll add parens, it's clearer indeed.